### PR TITLE
Make Fast404 faster

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

**Comment by AronNovak** [comment]
image
@amitaibu Would you make it fancier or is it a per-site thing to do for each client anyways?

File: web/modules/custom/server_general/tests/src/ExistingSite/ServerGeneralFast404Test.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+namespace Drupal\Tests\server_general\ExistingSite;
+
+use weitzman\DrupalTestTraits\ExistingSiteBase;
+
+/**
+ * Fast 404 test.
+ */
+class ServerGeneralFast404Test extends ExistingSiteBase {
+
+  const DRUPAL_META_IDENTIFY = '<meta name="Generator" content="Drupal 10 (https://www.drupal.org)" />';
```
Comment (AronNovak) [inline web/modules/custom/server_general/tests/src/ExistingSite/ServerGeneralFast404Test.php:12]:
```suggestion
  const DRUPAL_META_IDENTIFY = '<meta name="Generator" content="Drupal';
```

[the remark this answers is not in the document]
Comment (AronNovak) [reply web/modules/custom/server_general/tests/src/ExistingSite/ServerGeneralFast404Test.php]: Good point, I do not include the version number then simply.

**Comment by amitaibu** [comment]
> Would you make it fancier

In what way? i.e. what more would you add?

**Comment by AronNovak** [comment]
@amitaibu 
A dummy logo, centered and a Back link to the homepage, to `/`.

**Comment by amitaibu** [comment]
No need for the logo, but you can add a link back to the homepage

**Comment by AronNovak** [comment]
@amitaibu 
image
That's how it looks like.

**Review by amitaibu — APPROVED** [review]
:+1:

**Pull request merged**